hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/compact/HoodieFlinkCompactor.java

```diff
@@ -99,6 +94,12 @@ public void start(boolean serviceMode) throws Exception {
     }
   }
 
+  public void shutDown() throws Exception {
+    if (compactionScheduleService != null) {
```

Comment:
didn't check this code for a while. Did we support async compaction scheduling already?

Reply:
What did you mean for async compaction scheduling ?

Reply:
scheduling compaction on the batch job when the streaming job is ingesting data into the table.

Reply:
No, we do not support that ~